src/Elastic.Apm.AspNetCore/Extensions/TransactionExtentions.cs

```diff
@@ -1,12 +1,11 @@
 using System.Net.Mime;
-using Elastic.Apm.Api;
-using Elastic.Apm.Config;
 using Elastic.Apm.Logging;
+using Elastic.Apm.Model;
 using Microsoft.AspNetCore.Http;
 
 namespace Elastic.Apm.AspNetCore.Extensions
 {
-	public static class TransactionExtensions
+	internal static class TransactionExtensions
```

Comment:
I agree with your assessment:

> Probably no one uses it

So is it worth spending any more time on it?

Reply:
> So is it worth spending any more time on it?

Yes, because it's just about changing `internal` back to `public` and we have 1 less breaking change. That can be done very quickly.

Reply:
Unfortunately it's not as simple as changing `internal` back to `public` because the signature of `CollectRequestInfo` changed.

Reply:
Oh, yeah I see. As said [here](https://github.com/elastic/apm-agent-dotnet/pull/577#discussion_r345905598), I'd left it as it is. I know, the `IConfigurationReader` went away, but technically we could still pass that as a parameter.

Anyway, ok, let's make it `internal` and hope no one uses it. But could we try to avoid these things in the future please?

Reply:
Definitely.